### PR TITLE
EL-4241: Date Range Picker: "Missing locale data" error

### PR DESCRIPTION
**Pull request by JonathanToddMF**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-4241

#### Description of Proposed Changes

Injected LOCALE_ID into the date-formatter-pipe.ts and based this as the third parameter in formatDate

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4241-date-range-picker-locale-data/#/landing

#### Downstream Build(s)

[ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/view/Developer/job/caf~ux-aspects-micro-focus~EL-4241-date-range-picker-locale-data~CI/)

### CODE REVIEW

**Comment by JonathanToddMF** [comment]
Build Link - https://jenkins.swinfra.net/job/SEPG/job/UXAspects/view/Developer/job/UXAspects~UXAspects~EL-4241-date-range-picker-locale-data~CI/

**Comment by JonathanToddMF** [comment]
Languages checked:

- German
- French
- Spanish
- Hindi
- Arabic
- Polish

File: src/pipes/date-formatter/date-formatter.pipe.ts [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 import { formatDate } from '@angular/common';
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
 import { DateFormatter } from './date-formatter.type';
 
 @Pipe({
     name: 'formatDate'
 })
 export class DateFormatterPipe implements PipeTransform {
 
+    constructor(@Inject(LOCALE_ID) private locale: string) { }
```
Comment (donna-marijne) [inline src/pipes/date-formatter/date-formatter.pipe.ts:10]:
```suggestion
    constructor(@Inject(LOCALE_ID) private _locale: string) { }
```

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Looks good. We generally prefix private properties with an underscore in this project.

**Review by donna-marijne — APPROVED** [review]
Verification (repro steps from https://portal.digitalsafe.net/browse/EL-4241):
1. https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4241-date-range-picker-locale-data/#/components/date-time-picker#date-range-picker - :heavy_check_mark:
2. https://pages.github.houston.softwaregrp.net/sepg-docs-qa/caf_CI_ux-aspects-micro-focus_EL-4241-date-range-picker-locale-data/#/ui-components/input-controls#date-range-picker - :heavy_check_mark:

**Review by Roland-Penny — APPROVED** [review]
Looks good. Accepted

**Pull request merged**